filament/src/materials/antiAliasing/taa.mat

```diff
@@ -0,0 +1,280 @@
+material {
+    name : taa,
+    parameters : [
+        {
+            type : sampler2d,
+            name : color
+        },
+        {
+            type : sampler2d,
+            name : depth,
+            precision: high
+        },
+        {
+            type : float,
+            name : alpha
+        },
+        {
+            type : sampler2d,
+            name : history
+        },
+        {
+            type : mat4,
+            name : reprojection,
+            precision: high
+        },
+        {
+            type : float[9],
+            name : filterWeights
+        }
+    ],
+    variables : [
+        vertex
+    ],
+    domain : postprocess,
+    depthWrite : false,
+    depthCulling : false
+}
+
+vertex {
+    void postProcessVertex(inout PostProcessVertexInputs postProcess) {
+        postProcess.vertex.xy = postProcess.normalizedUV;
+    }
+}
+
+fragment {
+
+// accurate box clipping
+#define BOX_CLIPPING_ACCURATE   0
+// clamping instead of clipping
+#define BOX_CLIPPING_CLAMP      1
+// no clipping (for debugging only)
+#define BOX_CLIPPING_NONE       2
+
+#if defined(TARGET_MOBILE)
+#define BOX_CLIPPING            BOX_CLIPPING_ACCURATE
+#define USE_YCoCg               0
```

Comment:
Will any of these options be eventually enabled on mobile? Or just too expensive? Should these be part of the API as toggle or even as part of a quality setting?

Reply:
I think probably at some point yes, once we understand better what works.